src/test/regress/expected/multi_create_table.out

```diff
@@ -52,6 +52,9 @@ HINT:  Consider using hash partitioning.
  
 (1 row)
 
+-- Manually colocate tables lineitem and orders
```

Comment:
Has been updated

Reply:
Hmm, we're marking two append partitioned tables as colocated. This is definitely something that we don't support and seems very confusing now. 

Can't we make them `CoPartitioned` while loading the data? Is that because of the different shard ranges? If that's the case, we're doing even more hacky way. 

How hard is it to make those tables `CoPartitioned()` naturally in the sense that they have the same shard ranges and colocated placements?